Bluepill-cli/Bluepill-cli/Simulator/BPSimulator.m

```diff
@@ -129,6 +130,35 @@ - (NSString *)installApplicationWithHost:(NSString *)testHost withError:(NSError
         }];
         return nil;
     } else {
+        NSString *fileName = [[testHost lastPathComponent] stringByDeletingPathExtension];
+        NSString *tmpFilePath = [NSString stringWithFormat:@"%@/%@.json", NSTemporaryDirectory(), fileName];
+        NSDictionary *options = @{
+                                  kOptionsEnvironmentKey: @{
+                                          @"DYLD_LIBRARY_PATH" : [NSString stringWithFormat:@"%@/usr/lib/swift:%@/Frameworks", self.config.simRuntime.root, testHost],
+                                          @"DYLD_INSERT_LIBRARIES" : [BPUtils findExecutablePath:@BP_TEST_DUMPER_NAME],
```

Comment:
It is a bit weird that you're looking for a `.dylib` using `findExecutablePath`... I think we need better naming here. At least `findExecutableOrLibPath`... If you can come up with something better I'm all ears.